src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -47,6 +48,33 @@ public enum WebAuthenticationType
     }
 
     /// <summary>
+    /// The valid values for the -SslProtocol parameter for Invoke-RestMethod and Invoke-WebRequest
+    /// </summary>
+    [Flags]
+    public enum WebSslProtocol
+    {
+        /// <summary>
+        ///  No SSL protocol will be set and the system defaults will be used.
+        /// </summary>
+        Default = 0,
```

Comment:
I think we should be exactly `Default = SslProtocols.None`.

Reply:
I'm not sure how well that works with Flags. We have to define 0 for the default here and if CoreFX decides `none` is no longer `0` we lose our Default option.

Reply:
I see `handler.SslProtocols = (SslProtocols)SslProtocol;` so I belive ` Default = SslProtocols.None ` is more accurate.

Reply:
`SslProtocols` will always have a `0` too since it is a Flag as well. I'm weary of changing this one. Wehn I was researching I saw multiple asks for the `SslProtocols` to be cleaned up an `None` removed. it seems to be universally detested as a field name that is better defined as `SystemDefault`. Looking at the implementation of `SecurityProtocolType` they, too, erred on the side of using `0`.

https://github.com/dotnet/corefx/blob/ba0e14d957cd2fed3e1dec54d2c5f2a92a746c74/src/System.Net.ServicePoint/src/System/Net/SecurityProtocolType.cs#L12